src/hatti/map/utils.cljs

```diff
@@ -519,11 +600,14 @@
       (register-mapboxgl-mouse-events owner map event-chan id_string style)
       (set-mapboxgl-paint-property
        map id_string (get-style-properties style :normal :stops stops))
+      ;; add layer that acts as border for point type layers,
+      ;; otherwise remove the layer if it exists
       (if (= :point style)
         (add-mapboxgl-layer map id_string layer-type
                             :layer-id circle-border
                             :paint {:circle-color "#fff" :circle-radius 6})
         (when (.getLayer map circle-border) (.removeLayer map circle-border)))
+      ;;(show-hexbins map id_string geojson)
```

Comment:
is a comment? add space after ;;, remove parens. if it's commented code think we can remove